staff/views/orgs.py

```diff
@@ -178,7 +178,7 @@ def get_queryset(self):
             ]
             qs = qs.filter(qwargs(fields, q))
 
-        return qs
```

Comment:
If it's fairly straightforward to do it would be good to have tests for these `qs.distinct` changes, as it would be good to avoid doing it again in future.

Reply:
I would love to have this but I couldn't find a nice solution.  There's several ways that each list view can produce duplicates so then we'd need to either have a test that ran through all the use cases (most of which we're already testing), or add extra objects to the existing tests.  Neither of which are _terrible_, but they both feel really prone to a developer forgetting to add them, which is exactly how we ended up here!

I'll keep it in mind and see if anything stands out as a nice, obvious fix to this, because it would definitely be desireable to have 🤔